### PR TITLE
Updated FL, FApi and Loom version

### PR DESCRIPTION
**Pull request by PSGitHubUser1**
I did so you don't have to.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -8,11 +8,12 @@ archives_base_name = porting_lib
 # is filled in at build
 mod_version = 2.1.<patch>
 
+loom_version = 1.5.+
```
Comment (IThundxr) [inline gradle.properties]: What is this for? It's not doing anything since it's not even hooked up in the build.gradles

**Review by IThundxr — CHANGES_REQUESTED** [review]
Has this been tested?

**Comment by PSGitHubUser1** [comment]
> Has this been tested?

Nope. But it should work because I didn't coded anything, just updated versions..

**Comment by IThundxr** [comment]
> > Has this been tested?
> 
> Nope. But it should work because I didn't coded anything, just updated versions.. 

Doesn't matter, updates can have many changes some of which could cause unintended behaviour, anytime you bump dependencies you should make sure everything works and builds fine, also you need to test with quilt aswell to make sure whatever version you've picked is available on quilt

**Comment by PSGitHubUser1** [comment]
That's hard, I never tried quilt... Although I know that almost everything is same. Because Quilt supprts fabric mods

**Comment by PSGitHubUser1** [comment]
How do I test it, like run Intellij IDEA and then what?

**Comment by IThundxr** [comment]
> That's hard, I never tried quilt... Although I know that almost everything is same. Because Quilt supprts fabric mods

Quilt is not fully 1:1 with fabric especially when it comes to fapi/qfapi as usually qfapi isn't always updated to the latest fapi version right away and you need to check the version it provides and use that

**Comment by PSGitHubUser1** [comment]
> > That's hard, I never tried quilt... Although I know that almost everything is same. Because Quilt supprts fabric mods
> 
> Quilt is not fully 1:1 with fabric especially when it comes to fapi/qfapi as usually qfapi isn't always updated to the latest fapi version right away and you need to check the version it provides and use that

Oh ok.. I thought that many things would be same if it was designed to support Fabric mods...

**Comment by TropheusJ** [comment]
Versions will be bumped when they’re needed.

**Comment by PSGitHubUser1** [comment]
Ok. 😐👍

**Pull request closed without merging**